### PR TITLE
feat(build): add CI scripts and Dockerfile

### PR DESCRIPTION
**Pull request by suprovsky**

### CODE REVIEW

File: .github/workflows/build-docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Build Docker image and push it
+on:
+  push:
+    branches:
+      - 'master'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: "etf2l/${{ github.event.repository.name }}"
```
Comment (garrappachc) [inline .github/workflows/build-docker-image.yml:9]: Why the `etf2l` prefix?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
+FROM node:18 AS build
```
Comment (garrappachc) [inline Dockerfile:1]: Use `node:lts-alpine` image

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+RUN adduser -D static
+USER static
+WORKDIR /home/static
+
+COPY --from=build /usr/src/app/build/ ./
+
+LABEL org.opencontainers.image.version="1.0.0" \
+      org.opencontainers.image.title="docs-tf2pickup-org" \
+      org.opencontainers.image.base.name="ghcr.io/tf2pickup-org/:1.0.0" \
```
Comment (garrappachc) [inline Dockerfile:20]: Fix image name

**Review by garrappachc — CHANGES_REQUESTED** [review]

**Comment by garrappachc** [comment]
Also, scope of this PR should be **build**: `feat(build): add CI scripts and Dockerfile`

File: .github/workflows/build-docker-image.yml (hunk 1 above)
Comment (suprovsky) [reply .github/workflows/build-docker-image.yml:9]: my bad :D

**Review by garrappachc — APPROVED** [review]

**Pull request merged**